### ISSUE TITLE
[Card] Add .inverted variation to the card component

### ISSUE THREAD

**Issue by Qqwy** [Semantic-Org/Semantic-UI#3478]
**Steps to Reproduce**
1. Create `<div class="ui card inverted"></div>`

**Expected**
The Card should use white text on a dark background.

**Result**
The Card does not do this

**Comment by jokeronaldo** [comment]
Added ti 2.1.x milestone, nice suggestion :+1:

**Comment by Maxhodges** [comment]
are those milestones public somewhere?

**Comment by natiki** [comment]
You can check the milestones at: https://github.com/Semantic-Org/Semantic-UI/milestones

**Comment by baadc0de** [comment]
I've tried this in the latest React-Semantic-UI and it did not work. I would suspect that the selected Milestone is wrong or the React build does not properly forward the `inverted` style.

**Comment by Maxhodges** [comment]
Hi @baadc0de actually it's because there is no inverted style for cards in semantic-ui. Semantic-UI-React is actually CSS agnostic. That is, they don't provide the CSS at all. You need to load that on your own. But if you look, you'll find there is no inverted style for card in the Semantic-UI CSS. You'll need to create you own, as I've done for my site.

![image](https://cloud.githubusercontent.com/assets/957271/22180056/7614018a-e0a8-11e6-97d7-18a429c4ba46.png)

**Comment by baadc0de** [comment]
Well, this is a perfect ticket to request it then. @jokeronaldo would you still consider it for inclusion into SUI, and so could you verify into which milestone release of SUI this request would go? @Maxhodges would you consider making a PR for this with your own mods to SUI?

**Comment by Maxhodges** [comment]
I just did it in my custom theme by overriding the card view. I used a 'dark' class instead of inverted (in case SUI makes an inverted style later). This might help you get started, but it might not be the whole picture because I think we override default text colors too in another place

blackship/client/lib/themes/blackship/views/card.overrides.import.less

```css
.ui.dark.cards > .dark.card, .ui.dark.card {
  max-width: 100%;
  position: relative;
  display: flex;
  flex-direction: column;
  width: 290px;
  min-height: 0px;
  background: @dark-bg-40;
  padding: 0em;
  border: none;
  border-radius: 0.28571429rem;
  box-shadow: 0px 1px 3px 0px @dark-bg-40, 0px 0px 0px 1px @dark-bg-40;
  transition: box-shadow 0.1s ease, transform 0.1s ease;
  z-index: '';
}

.ui.dark.card {
  background: @dark-bg-40;
}

.ui.dark.cards > .dark.card > .content > a.header, .ui.dark.card > .content > a.header {
  color: white;
}

.ui.cards > .card .meta, .ui.card .meta {
  font-size: 1em;
  color: @grey-50;
}

.ui.dark.cards > .dark.card > .content > .header, .ui.dark.card > .content > .header {
  //display: block;
  //margin: '';
  //font-family: 'Roboto', 'sans-serif';
  color: white;
}

.ui.dark.cards > .dark.card > .content > .description, .ui.dark.card > .content > .description {
  clear: both;
  color: white;
}

.ui.dark.cards > .dark.card > .content > a.header:hover, .ui.dark.card > .content > a.header:hover {
  color: rgba(0, 0, 0, .8);
}

.ui.dark.cards > .dark.card > .extra,
.ui.dark.card > .extra {
  border-top: 1px solid @textColor !important;
  color: inherit;
}

//@shadowHoverBoxShadow: none;
//@linkHoverBorder: 1px;
```

**Comment by baadc0de** [comment]
Thanks, it certainly helps! 👍

**Comment by VSG24** [comment]
Any updates on this??

**Comment by lubber-de** [comment]
The community fork Fomantic-UI implemented a inverted card variant since 2.5
https://fomantic-ui.com/views/card.html#inverted

**Comment by Qqwy** [comment]
Thank you very much for pointing me to Formantic-UI!.
